### DISCUSSION TITLE
Editing content models in the dashboard

### DISCUSSION THREAD

**Discussion by vimtor** [swellstores/community#158] [Ideas]
Hi there!

I've created a new model called `BlogPost`. I expected it to appear under some kind `Content` sidebar item. I guess this is not a feature but it will be cool if I could add custom content types and manage them in the dashboard for CMS purposes.

I know that there are tools like Contentful or Sanity.io that could get the job done but I feel that for small stores this could be overkill. Also, since you already have the backend part (custom modeling) I guess it could be easy to generate the forms programmatically in the dashboard frontend using the field types.

**Comment by ericingram** [comment]
Agree! We are actively working on this now 🚀

**Comment by lossaloss** [comment]
Hi, we are also interested in this feature. Any timeframe for when this will be available?
